src/ExpressReceiver.ts

```diff
@@ -100,11 +100,8 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
     };
 
     if (req.body && req.body.response_url) {
-      event.respond = (response): void => {
-        this.axios.post(req.body.response_url, response)
-          .catch((e) => {
-            this.emit('error', e);
-          });
+      event.respond = async (response) => {
+        await this.axios.post(req.body.response_url, response);
```

Comment:
I removed the `catch` so this will behave more like promises would expect. Left unchecked in the new middleware pattern, it will be bubbled up to the global error handler _anyway_ I figured this was safe and more expected.

Reply:
Update: I'm going to update this pr to move this block into onIncommingEvent so that every receiver doesn't need to implement it. 

Incoming shortly.